torch/mps/__init__.py

```diff
@@ -140,13 +140,13 @@ def recommended_max_memory() -> int:
     return torch._C._mps_recommendedMaxMemory()
 
 
-def _compile_shader(source: str):
```

Comment:
Should we have more detailed doc on the argument conversion since it is not the same as our usual python bindings?

Reply:
Will do, in a follow up PR (want to technically land this one before branch cut)